dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -200,3 +200,7 @@ set400SCOnTooManyParentDirs.desc=When this property is set to true, a request wi
 
 allowAbsoluteFileNameForPartWrite.name=Write part files to absolute file locations
 allowAbsoluteFileNameForPartWrite.desc=When this property is set to true, the runtime treats the filename argument on the part.write method call as an absolute location, rather than saving the filename argument under the temporary location. In servlet-6.1 and later, the default value is true. In servlet-6.0 and earlier, the default value is false. 
+
+maxPartHeaderSize.name=The maximum size of a single part headers 
```

Comment:
Remove the plural s from headers:

maxPartHeaderSize.name=The maximum size of a single part header
maxPartHeaderSize.desc=The maximum size of the header for a single part in bytes. The default value is 512. For unlimited size, set the value to -1.

Reply:
@is273 size of all the headers (in a single part)

Reply:
@pmd1nh So then:

maxPartHeaderSize.name=The maximum size of the single part headers
maxPartHeaderSize.desc=The maximum size of the headers for a single part in bytes. The default value is 512. For unlimited size, set the value to -1.